packages/ui/src/server/components/vcs/index.ts

```diff
@@ -33,7 +26,7 @@ export class VcsFactory {
             case 'gitlab':
                 return new GitlabApi(apiConfig);
             default:
-                throw new ErrorWithCode(415, `Unsupported VCS type: ${vcs.type}`);
+                throw new ErrorWithCode(415, `Unsupported VCS type`);
```

Comment:
> HTTP client error response code 415 Unsupported Media Type indicates that the server refuses to accept the request because the content format is not supported by the server.

I think 415 is more appropriate here